app/validators/wpcc/salary_threshold_validator.rb

```diff
@@ -0,0 +1,22 @@
+module Wpcc
+  class SalaryThresholdValidator < ActiveModel::Validator
+    SALARY_THRESHOLDS  = [5876.00, 489.67, 452.00, 113.00].freeze
+    SALARY_FREQUENCIES = %w[year month fourweeks week].freeze
+    LOW_SALARY_THRESHOLDS = SALARY_FREQUENCIES.zip(SALARY_THRESHOLDS).to_h
```

Comment:
Or other solution:

A solution to this:

On the validator:

```ruby
class SalaryThresholdValidator 
  SALARY_FREQUENCIES = 
   YAML.load_file(Wpcc::Engine.root.join('config','salary_frequencies.yml'))
end
```

On the form:
```ruby
class YourDetailsForm
  SALARY_FREQUENCIES = 
   YAML.load_file(Wpcc::Engine.root.join('config','salary_frequencies.yml'))
end
```

@willhall88 & @Tr4pSt3R wdyt?

I will rename the PR to WIP since everybody agrees that needs to be changed. We just need to agree what is the change itself.